sources/packages/backend/apps/api/src/route-controllers/application-exception/_tests_/e2e/application-exception.aest.getExceptionById.e2e-spec.ts

```diff
@@ -0,0 +1,102 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
+import * as request from "supertest";
+import { Repository } from "typeorm";
+import {
+  AESTGroups,
+  BEARER_AUTH_TYPE,
+  createTestingAppModule,
+  getAESTToken,
+} from "../../../../testHelpers";
+import {
+  createFakeApplicationException,
+  createFakeApplicationExceptionRequest,
+} from "@sims/test-utils";
+import {
+  ApplicationException,
+  ApplicationExceptionRequest,
+  ApplicationExceptionStatus,
+  User,
+} from "@sims/sims-db";
+import { ClientTypeBaseRoute } from "../../../../types";
+import { SystemUsersService } from "@sims/services";
+import { getUserFullName } from "../../../../utilities";
+
+describe(`${ClientTypeBaseRoute.AEST}-ApplicationExceptionAESTController(e2e)-getExceptionById`, () => {
+  let app: INestApplication;
+  let applicationExceptionRepo: Repository<ApplicationException>;
+  let applicationExceptionRequestRepo: Repository<ApplicationExceptionRequest>;
+  let systemUsersService: SystemUsersService;
+  let userRepo: Repository<User>;
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource, module } =
+      await createTestingAppModule();
+    app = nestApplication;
+    applicationExceptionRepo = dataSource.getRepository(ApplicationException);
+    applicationExceptionRequestRepo = dataSource.getRepository(
+      ApplicationExceptionRequest,
+    );
+    systemUsersService = await module.get(SystemUsersService);
+    userRepo = dataSource.getRepository(User);
+  });
+
+  it("Should get an application exception by id when available.", async () => {
```

Comment:
does `when available.` means "pending status?

Reply:
"pending status" can be expressed in the E2E description also to be clear.

Reply:
@andrepestana-aot I see that you acknowledged @ann-aot 's comment. I was not able to follow that, as the API does not get only pending exceptions by exceptionId as per the logic I see. Am I missing something here?

Reply:
In this case it is "when available" because as @dheepak-aot said it can be application exceptions with any status. I didn't change the test for this comment.

Reply:
@dheepak-aot I was not clear about it, if @ann-aot was right, I would like to ensure that the comment was more precise. I did not look into the controller code.